tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -5953,3 +5956,308 @@ def get_prod(arr):
 
     assert (f_complex64(cmplx64) == get_prod(cmplx64))
     assert (f_complex128(cmplx128) == get_prod(cmplx128))
+
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = [pytest.mark.fortran]),
+        pytest.param("c", marks = [
+            pytest.mark.skip(reason="Still under maintenance, See #769"),
+            pytest.mark.c]
+        ),
+        pytest.param("python", marks = [pytest.mark.python])
+    )
+)
+
+def test_numpy_norm_scalar(language):
+
+    @types('bool')
+    @types('int')
+    @types('int8')
+    @types('int16')
+    @types('int32')
+    @types('int64')
+    @types('float')
+    @types('float32')
+    @types('float64')
+    def get_norm(a):
+        from numpy.linalg import norm
+        b = norm(a)
+        return b
+
+    @types('complex64')
+    @types('complex128')
+    def get_norm_complex(a):
+        from numpy.linalg import norm
+        b = norm(a)
+        return b
```

Comment:
You have split the tests in the scalar case, but not in the array cases and you have also added logic to the code so you don't need to split the cases. Please handle this the same way in all the tests